.env.development

```diff
@@ -52,10 +52,11 @@ ENABLE_FIXED_OTP_ON_REQUEST_FOR_QA=false
 ENABLE_GENERATE_ENCOUNTER_ID_ENDPOINT=true
 ENABLE_PATIENT_FOLLOWUPS=true
 ENABLE_PURGE_ENDPOINT_FOR_QA=false
-ENABLE_SMS_REMINDERS=true
+ENABLE_APPOINTMENT_REMINDERS=true
```

Comment:
I guess we'll have to _add_ `ENABLE_APPOINTMENT_REMINDERS`, release this PR, then _remove_ `ENABLE_SMS_REMINDERS`, right? Just calling it out 👍

Reply:
You're right :)